src/OpenSpaceToolkit/Astrodynamics/Trajectory/State.cpp

```diff
@@ -354,10 +356,25 @@ void State::print(std::ostream& anOutputStream, bool displayDecorator) const
     ostk::core::utils::Print::Line(anOutputStream)
         << "Instant:" << (this->instant_.isDefined() ? this->instant_.toString() : "Undefined");
     ostk::core::utils::Print::Line(anOutputStream)
-        << "Position:" << (this->isDefined() ? this->getPosition().toString(12) : "Undefined");
-    ostk::core::utils::Print::Line(anOutputStream)
-        << "Velocity:" << (this->isDefined() ? this->getVelocity().toString(12) : "Undefined");
+        << "Frame:" << (this->frameSPtr_->isDefined() ? this->frameSPtr_->getName() : "Undefined");
 
+    if (!this->isDefined())
+    {
+        ostk::core::utils::Print::Line(anOutputStream) << "Coordinates: Undefined";
+    }
+    else
+    {
+        const Array<Shared<const CoordinatesSubset>> subsets = this->coordinatesBrokerSPtr_->getSubsets();
+
+        ostk::core::utils::Print::Line(anOutputStream) << "Coordinates: " << this->accessCoordinates().toString(4);
```

Comment:
```suggestion
```
I think that we can skip printing the coordinates since we're printing it piece by piece below.

Reply:
My thinking was that it could be nice in case we wanted to copy-pasta the coordinates, but looks like we didn't have that before...  Going to close this suggestion and implement this locally so I can also remove the "Where" line

Reply:
(implemented)